content/_index/contact.md

```diff
@@ -0,0 +1,51 @@
++++
+fragment = "contact"
+#disabled = true
+date = "2017-09-10"
+weight = 1100
+#background = "light"
+form_name = "defaultContact"
+
+title = "Contact fragment"
+subtitle  = "*not working on demo page*"
+
+# PostURL can be used with backends such as mailout from caddy
+post_url = "https://example.com/mailout" #default: formspree.io
+email = "mail@example.com"
+button = "Send Button" # defaults to theme default
+#netlify = false
+
+# Optional google captcha
+#[recaptcha]
+#  sitekey = ""
+
+[message]
+  success = "Thank you for awesomely contacting us." # defaults to theme default
+  error = "Message could not be send. Please contact us at mail@example.com instead." # defaults to theme default
```

Comment:
Not sure, but as we have a default, we could make these messages empty, but keep them for someone to add in.

success = ""
error = ""